### ISSUE TITLE
Some expected weirdness in visuals

### ISSUE THREAD

**Issue by terracoda** [phetsims/quadrilateral#268]
I know @jessegreenberg is making big changes right now to the code, so these examples may not last long, but just noting them as I do testing in case that is helpful.

Is that a good approach?
<img width="778" alt="image" src="https://user-images.githubusercontent.com/1902116/205701718-fb72a7d7-021e-48bb-b969-ba3773aea510.png">

**Comment by terracoda** [comment]
Actually sim crashed after that. I'll try my dev version rather testing off master.

**Comment by terracoda** [comment]
Don't have show model value son, but here I was moving down and the side took off on its own to quickly collapse the shape and then the sim crashed - nothing operable.
<img width="738" alt="image" src="https://user-images.githubusercontent.com/1902116/205706307-239e94fe-b468-49fe-8748-f6253f47cd89.png">

**Comment by jessegreenberg** [comment]
Yes, thank you for reporting! Lots of big churn in the next couple of weeks so I expect things may get unstable. Ill take a look at what is causing this crash.

**Comment by terracoda** [comment]
Corner C got stuck with Keyboard in this shape, but after using the mouse, it was fine again.  
<img width="851" alt="image" src="https://user-images.githubusercontent.com/1902116/206291354-3421ea9f-73b0-4f4a-b3e9-7694f7b27170.png">

**Comment by terracoda** [comment]
I was maintaining a trapezoid moving AM to the right and the shape broke, and the sim became inoperable. Had to do a page refresh.

<img width="849" alt="image" src="https://user-images.githubusercontent.com/1902116/206474633-5d35217b-83ea-4358-8bf8-6257e94effb3.png">

**Comment by jessegreenberg** [comment]
I am not sure exactly what is going on in these screenshots, but I recently fixed #266, https://github.com/phetsims/quadrilateral/issues/280 and https://github.com/phetsims/quadrilateral/issues/279. I have been letting the sim fuzz with ?fuzz and ?fuzzBoard and I have not seen any crashing.  I am howerver still seeing some vertex stickiness. That has its own issue #271, so Ill continue there. Closing this issue.
